server/server.py

```diff
@@ -107,14 +74,29 @@ def do_GET(self):
         path = parsed_url.path
 
         if path == '/init':
-            commands = unquote(params['commands'][0]).split(',')
-            size = int(params['size'][0])
             sample_rate = int(params['sampleRate'][0])
-            result = init_audio_files(commands, size)
+            command_list = unquote(params['commands'][0]).split(',')
+            seed = int(params['randomSeed'][0])
+            X, Y = prepare_dataset()
+            p = np.random.RandomState(seed).permutation(len(X))[int(.9 * len(X)):]
+            X_test = X[p]
```

Comment:
The validation and test sets will need to be the same as the ones used in the official dataset, e.g., `validation_list.txt`.